include/biosHandler.hpp

```diff
@@ -47,7 +113,19 @@ class BiosHandler
      */
     void checkAndListenPldmService();
 
+    /**
+     * @brief Register listener for BIOS attribute property change.
+     *
+     * The VPD manager needs to listen for property change of certain BIOS
+     * attributes that are backed in VPD. When the attributes change, the new
+     * value is written back to the VPD keywords that backs them up.
+     */
+    void listenBiosAttributes();
```

Comment:
listenBiosAttributesChanges ?

Reply:
listenOnBiosAttributes ?

Reply:
We agreed to continue with the same API name.